packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -866,7 +867,6 @@ contract VotingV2 is
                 continue;
             }
 
-            if (requestIndex > indexTo - 1) break; // This happens if the last element was a rolled vote.
```

Comment:
This was the change I was a little unsure about.

I think this was added in another fix, but looking at it again, I have no idea how this could ever happen.

There is no way that requestIndex could be modified in this iteration except in the continue (which means we would skip this statement).

If it was modified in a previous iteration, the `requestIndex < indexTo` should cause the loop to break before getting here.

Happy to add this back, but I'm not sure how this could ever be hit.

Reply:
ye, you right. I think this was residual after a previous change pre-audit.

Reply:
Agree, I think this was due to the `indexTo++` on rolled votes that we have removed.